metric.go

```diff
@@ -290,3 +247,30 @@ func ValidateTags(tags []string) bool {
 
 	return true
 }
+
+func writeSortedTagString(w io.Writer, name string, tags []string) error {
+	sort.Strings(tags)
+
+	_, err := io.WriteString(w, name)
+	if err != nil {
+		return err
+	}
+
+	for _, t := range tags {
+		if len(t) >= 5 && t[:5] == "name=" {
```

Comment:
```suggestion
		if len(t) > 5 && t[:5] == "name=" {
```
So that it matches the other change in NameWithTags()? I believe there may be more instances of this.

Reply:
actually a tag value of `len(value)<1` is invalid. so `len(5) > 5` makes sense, because with a value length of `>=1` the total length of `name=<value>` must be at least `6`